src/python/arcor2_arserver/scripts/arserver.py

```diff
@@ -186,6 +186,7 @@ async def register(websocket: WsClient) -> None:
 
 
 async def unregister(websocket: WsClient) -> None:
+    await glob.LOCK.release_all_owner_locks("ui.owner")
```

Comment:
Co když uživatel pustí nějaký dlouho trvající pohyb robota a potom uspí tablet - AREditor se v takovém případě odhlásí od serveru. Zůstane potom ten robot zamčený do dokončení pohybu?

Reply:
Dobrá připomínka. 
Otázkou je, která RPC aktivují robota. Jedná se jen o rpc `execute_action_cb`? Nebo například i něco z robot.py?

Reply:
Pokud by se jednalo jen o tohle RPC, tak stačí to asynchronní spuštění převést pod server name konstantu, to by řešilo problém s odhlášením. Otázkou je, jestli je to odolné vůči tomu lock timeoutu (5 minut)?

Reply:
Přes `execute_action_cb` jde pustit jakákoliv akce, libovolného objektu - i některé jiné než "robotí" akce mohou trvat dlouho. Pohyb robota jde potom vyvolat i jinak než přes akci - move_to_pose apod. z robot.py.